ra2ce/analyses/analysis_config_data/analysis_config_data_validator.py

```diff
@@ -56,7 +56,7 @@ def _validate_header(self, header: Any) -> ValidationReport:
                     continue
                 if isinstance(value, Ra2ceEnumBase):
                     # enumerations
-                    _expected_values_list = value.list_valid_options()
+                    _expected_values_list = type(value).list_valid_options()
```

Comment:
This should not be needed when working with a classmethod, the object itself will still have access to it.